.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,68 @@
+---
+name: 🐛 Bug Report
+about: Report erros and unexpected behavior
+title: 'My descriptive bug title'
+labels: ''
+assignees: ''
+
+---
+
+<!--
+
+Thank you for using Pester and taking the time to report this issue!
+
+Please be aware that Pester version 3.4.0 - which is shipped with Windows 10 and Windows Server 2016 - is not supported anymore.
```

Comment:
This can be removed. Updating is mentioned below.

Reply:
Full stop on v4 bugs reports too? "Please update Pester" was referring to latest v4 or v5, however v3 is done - won't be fixed.